quesma/clickhouse/parser.go

```diff
@@ -146,6 +146,8 @@ func SchemaToColumns(schemaMapping *schema.Schema, nameFormatter TableColumNameF
 			fType = "Nullable(DateTime64)"
 		case schema.TypeDate.Name:
 			fType = "Nullable(Date)"
+			// TODO if someone sets `type: date` to a field in schemaOverrides AND this is a timestamp field for which we have dedicated logic (use DateTime64 + add DEFAULT now64())
```

Comment:
NOT, but TODO are mostly used in form `TODO:`

Reply:
Not sure also if I understand `TODO`, they should be in form what is to be fixed and explanation.